antora-ui-camel/src/css/img.css

```diff
@@ -0,0 +1,8 @@
+article h1.page img {
+  width: 1em;
+}
+
+.nav-link .image > img,
+.breadcrumbs li .image > img {
+  width: 1em;
+}
```

Comment:
Not sure about this change, is it related to catalog? Should go to the `catalog.css` and scoped on `.catalog`.

Reply:
No actually in the description pages (which are using the `default` layout, not the `catalog` one) icon images in the links in the left column are displayed full-size without this. The same happen for the icon in the title. I've put these general rules in a separate `img.css` file because they are not strictly related to the catalog.

Reply:
Then the `.nav-link...` change needs to go to `nav.css` and `article h1.page img` should go to `doc.css` and scoped with `.doc` (so `.doc article h1.page img`).